app/models/submission_completion.rb

```diff
@@ -24,4 +28,8 @@ def perform!
   def create_invoice_for?(submission)
     !submission.report_no_business? && submission.total_spend != 0 && ENV['SUBMIT_INVOICES']
   end
+
+  def mark_old_submissions_as_replaced!
```

Comment:
I think that if we are saying long-term (i.e. once the data migration has tidied up the legacy interventions) there will only ever be one `completed` submission, then this method should only mark the one `completed` submission as replaced. Otherwise it leaves the suggestion in the codebase that there could be a state where multiple completed submissions might exist, which could confuse and mislead. i.e. rename to `mark_completed_submission_as_replaced` and use `find` instead of `where`.

Reply:
OK, just noticed that this was done after feedback from @leeky's review! Re: this being done to prevent data getting weird if someone messes something up in the console, I would say that if our intention is that the replacement returns feature work will make console messing to replace returns redundant, then we shouldn't add code to protect against someone doing it wrong in the console. If we still need to mess around in the console after shipping this feature, then I'd say we've done something wrong.

Reply:
We've gone back and forth on this. That's what we had originally, and leeky suggested in this PR we do it this way based on his experience, that we wanted to be *extra sure* that there is only ever one completed submission, which is why we re-wrote it like this.

I suppose it's a question of do we write the code assuming we have the data state we want or write it more defensively than that. I'm easy either way.

Reply:
I generally err on the side of assuming that the system is in a good state, because otherwise you could be masking problems elsewhere. If there is a worry about the system getting into a bad state, then I would say it would be better to spend the time to prevent that in the first place, rather than adding code to handle it.

Reply:
Fair enough, we've updated it back. Good to merge?